src/pages/DeviceList.js

```diff
@@ -8,51 +8,51 @@ import VolumeSlider from '../components/VolumeSlider'
 async function setConfiguration (params) {
   const body = JSON.stringify(params)
   const headers = { 'Content-Type': 'application/json' }
-  await fetch('/api/configuration', { method: 'PATCH', body, headers })
+  await fetch('/api', { method: 'PUT', body, headers })
 }
 
-async function getDevices () {
+async function getConfiguration () {
   console.log('fetching!')
-  const res = await fetch('http://localhost:3000/api/devices')
+  const res = await fetch('/api')
   const body = await res.json()
   console.log(body)
   return body
 }
 
 function DeviceList () {
+  const [device, setDevice] = useState(null)
   const [volume, setVolume] = useState(30)
   const [devices, setDevices] = useState([])
 
-  useEffect(() => {
-    getDevices().then(setDevices)
-    const id = setInterval(async () => {
-      console.log('fetching')
-      getDevices().then(setDevices)
-    }, 5000)
+  async function update () {
+    const { device, volume, devices } = await getConfiguration()
+    setDevice(device)
+    if (volume !== null) setVolume(volume)
```

Comment:
We should probably update the server to start with a default volume, so that this is never null...